### PR TITLE
starting on post 8081 + integration with weather service

### PR DESCRIPTION
**Pull request by yuriyKolomiyets**

### CODE REVIEW

File: src/main/java/com/example/travelservice/integration/WeatherFromApi.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class WeatherFromApi {
```
Comment (bizianov) [inline src/main/java/com/example/travelservice/integration/WeatherFromApi.java]: please rename it, all services should end on "Service", for example WeatherIntegrationService

File: src/main/java/com/example/travelservice/integration/WeatherFromApi.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    private final String host = "http://localhost:8080/api/";
```
Comment (bizianov) [inline src/main/java/com/example/travelservice/integration/WeatherFromApi.java]: please extract it to the property file

**Review by bizianov — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/travelservice/integration/WeatherFromApi.java (hunk 2 above)
Comment (yuriyKolomiyets) [reply src/main/java/com/example/travelservice/integration/WeatherFromApi.java]: done

File: src/main/java/com/example/travelservice/integration/WeatherFromApi.java (hunk 1 above)
Comment (yuriyKolomiyets) [reply src/main/java/com/example/travelservice/integration/WeatherFromApi.java]: done

**Review by bizianov — APPROVED** [review]

**Pull request merged**